### PR TITLE
Make system tests pass when run locally on non-English systems

### LINKED ISSUE #13362: When running tests locally, NVDA depends on the system language to be English
### Steps to reproduce:

1. On a system that is not set to English as the language, but has a language supported by NVDA, build NVDA from source.
2. Run tests, at least the system tests.

### Actual behavior:

Almost all tests fail because the NVDA test harness expects English language output everywhere, including the Chrome UI. Local tests cannot be run and fail almost 100%.

### Expected behavior:

NVDA should force itself and the apps it uses to run in English if it needs that, even on non-English systems.

#### NVDA installed/portable/running from source:

Running from source.

#### NVDA version:

Any current master that supports running tests.

#### Windows version:

Any supported.

#### Name and version of other software in use when reproducing the issue:

Chrome browser for system robot Chrome tests.

#### Does the issue still occur after restarting your computer?

Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No.

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

N/A.

### PR DESCRIPTION
**Pull request by CyrilleB79**
### Link to issue number:
Closes #13362

### Summary of the issue:
The majority of system tests fail when run on non-English systems. On my system (French), failures are due to the following reasons:
- Default NVDA's UI language is user's default Windows language, thus the messages of NVDA are reported in this language.
- If not specified, the language of an HTML page is set by the browser itself which may not be English (system language?)
- According to the locale keyboard layout, someemulated  keypress may fail. E.g. "NVDA+\" cannot be emulated with French keyboard layout since \ (backslash) can only be accessed with modifiers (AltGr).

In #13362, it was also stated that Chrome's language may make some tests fail. That's not the case on my French system. But we may imagine that the keyboard shortcuts could vary from one language to another, e.g. "Alt+d" to focus the address field.

### Description of user facing changes
System tests now pass on French system. It should pass on any system regardless of its language.

### Description of development approach

1. NVDA interface language is forced to English in the various config files.
2. HTML sample content forced to English (via lang HTML attribute)
3. Removed the unneeded NVDA+\ custom gesture which cannot be executed without modifier on some keyboard layout (e.g. French). Actually, there is no need for custom gesture since `NVDA+d` is now a native gesture of NVDA for the same script.
   Even if not used anymore for now, I have not removed the possibility to use a custom gesture file in case an unassigned script has to be tested in the future. Should it be the case, care should be taken to assign a gesture that can be executed on any keyboard layout, i.e. use a letter rather than punctuation as the main key name.
E. Chrome UI language is forced to English via command line parameter (see [List of Chromium Command Line Switches « Peter Beverloo](https://peter.sh/experiments/chromium-command-line-switches/))

Note: For the Chrome command line --lang parameter to be taken into account, it should be ensured that no previous Chrome window is open when the tests are run.

### Testing strategy:

- [x] System tests pass on my French local system
  Windows 10 20H2 (x64) build 19042.1889, Chrome 105
  Only "NVDA" tag tested, not "installer".:
  `runsystemtests.bat -i nvda`
- [x] System test pass locally on an English system (tested by @lukaszgo1 on Windows 10)
- [x] System tests pass on appVeyor

### Known issues with pull request:

I have not the opportunity to test it myself.
But according to @lukaszgo1's tests, Notepad system test fail on Windows 11.
Since it is unrelated to the changes of this PR, I suggest to handle in a different issue/PR if needed.

### Change log entries:
Bug fixes (or For Developers ?)
`System test should now pass when run locally on non-English systems. (#13362)`

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] Security precautions taken.

Cc @lukaszgo1, @MarcoZehe

### CODE REVIEW

**Comment by CyrilleB79** [comment]
If someone would like to test with an English system, this would be helpful.

**Comment by lukaszgo1** [comment]
@CyrilleB79  Many thanks for taking this! Ive tested and on my English Windows 10 21H2 all system tests passes.
Unfortunately on my Polish Windows 11 all notepad tests fail - it is hard to say if this is caused by this PR, since previously it was impossible to test on non-English systems at al. Would someone running English Windows 11, perhaps @josephsl  be able to check if Notepad system tests from current master work for them?
While this PR is certainly a pretty big improvement for contributors running non-English Windows and should be merged, it should also be noted that to enable testing NVDA in different languages (desirable as mentioned i.e. in #12710) a more sophisticated solution may need to be written in the future.

File: tests/system/robot/chromeTests.robot [hunk 1]
```diff
@@ -24,7 +24,7 @@ default teardown
 	quit NVDA
 
 default setup
-	start NVDA	standard-dontShowWelcomeDialog.ini	chrome-gestures.ini
+	start NVDA	standard-dontShowWelcomeDialog.ini
```
Comment (seanbudd) [inline tests/system/robot/chromeTests.robot:27]: should the file `chrome-gestures.ini` be deleted?

**Review by seanbudd — COMMENTED** [review]
Generally LGTM, one minor suggestion

File: tests/system/robot/chromeTests.robot (hunk 1 above)
Comment (CyrilleB79) [reply tests/system/robot/chromeTests.robot:27]: Yes. Done in a200a9c36ca82634bba1a583bcdc4cc741cec198.

It was actually my intention but I have forgotten it.

**Comment by CyrilleB79** [comment]
@lukaszgo1, could you have a look or provide the logs of one of the failing Notepad test to be able to investigate why it fails and if it can be fixed?

Converting to draft, waiting for a test on English Windows 11 (on this PR's branch and on last alpha).

**Comment by CyrilleB79** [comment]
Ping @lukaszgo1, @josephsl:
* @josephsl would you be able to test this on Windows 11? If not (due to lack of time or any other reason), just let me know so that I ask to someone else, e.g. on nvda-devel
* @lukaszgo1 would you mind provide information on the failing tests?
Many thanks!

**Comment by lukaszgo1** [comment]
I haven't, yet, analyzed the logs from the failed tests, however I've switched my Windows 11 to English and can confirm that Notepad tests fail even on the current master. so the problem has not been introduced in this PR. Sorry for the confusion.

**Comment by CyrilleB79** [comment]
> I haven't, yet, analyzed the logs from the failed tests,

Feel free to provide the log results if you want me to have a look.

> however I've switched my Windows 11 to English and can confirm that Notepad tests fail even on the current master. so the problem has not been introduced in this PR.

Thanks for this valuable test and information.

Since the remaining issue is not directly linked to this PR nor to the corresponding issue (#13362), I set this PR ready for review.

**Review by seanbudd — APPROVED** [review]

**Comment by seanbudd** [comment]
I think the change log entry should be in "for developers" or not at all.
I don't think changes that only affect developers should go in "bug fixes".

**Pull request merged**